src/components/ListingPage/MobileFilterSection.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="flex flex-col">
+    <Label
+      :text="$t('listing_page_filter')"
+      align="center"
+      class="mb-[12px] text-[18px] font-600"
+    />
+    <div class="w-full border-b-[1px] border-shade-gray my-[8px]" />
+    <ListingPageFilterSection
```

Comment:
Add a prop for mobile and dont use CSS breakpoint
```suggestion
    <ListingPageFilterSection
      :is-mobile="true"
```

Reply:
[💄 Add ListingPageDialog](https://github.com/likecoin/liker-land/pull/1612/commits/389a25a0a96802ab4170e10d5af15e91e5e1c63c)